reward_index/src/indexer.rs

```diff
@@ -130,7 +130,14 @@ impl Indexer {
                             key: PublicKeyBinary::from(r.hotspot_key).to_string(),
                             reward_type: RewardType::MobileGateway,
                         },
-                        r.dc_transfer_reward + r.poc_reward,
+                        r.poc_reward,
+                    )),
+                    Some(MobileReward::DataTransferReward(r)) => Ok((
```

Comment:
Yeah i kinda agree with the above statement.  The IOT side does feel more correct to me as the rewards are targeted at either a gateway or a fund.  The gateway rewards are made up of beacons, witnesses and data transfer.   From an organisational perspective having the 3 amounts in the one record/proto ensures that for any one rewards epoch only a single file needs consulted to determine the total rewards for any one gateway.  This is true both for the indexer oracle itself and also from a data analysis perspective.

With the approach in this PR, there are now two files that have to be consulted in order to determine the full rewards for any one gateway.  I am not sure i see any added value in that ?  If you can explain the driver behind this separation it will help to make things clearer I imagine.

If the driver is that we are rewarding the Mobile Radio and the Gateway seperately and these both have different public keys then I can see the need for this, but i dont think that is the driver as that would mean all rewards to date are incorrect.